contracts/eosio.wrap/src/eosio.wrap.cpp

```diff
@@ -2,15 +2,14 @@
 
 namespace eosio {
 
-void wrap::exec( ignore<name>, ignore<transaction> ) {
+void wrap::exec( name executer, transaction trx) {
    require_auth( get_self() );
-
-   name executer;
-   _ds >> executer;
-
    require_auth( executer );
 
-   send_deferred( (uint128_t(executer.value) << 64) | (uint64_t)current_time_point().time_since_epoch().count(), executer, _ds.pos(), _ds.remaining() );
+   // Inline execution of the wrapped transaction.
```

Comment:
Also assert that there are no context free actions.

Reply:
https://github.com/EOSIO/eosio.contracts/blob/deprecate-deferred3/contracts/eosio.wrap/src/eosio.wrap.cpp#L9